app/database/operator/server_data_operator/handlers/post.test.ts

```diff
@@ -657,6 +657,56 @@ describe('*** Operator: Post Handlers tests ***', () => {
         expect(spyOnBatchRecords).toHaveBeenCalledWith(scheduledPosts, 'handleScheduledPosts');
     });
 
+    it('HandleUpdateScheduledPostErrorCode: should update error code for a scheduled post', async () => {
```

Comment:
0/5 if we want to test also for `prepareRecordsOnly = true`.

Reply:
Have created a issue to track this one: https://mattermost.atlassian.net/browse/MM-63461